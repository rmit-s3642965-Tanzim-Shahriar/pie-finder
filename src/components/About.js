import React, { Component } from 'react';
import './about.scss';
class About extends Component
{
    render()
    {
        return(
            <div className = 'about'>
                <div className = 'headerTitle'>WHAT DOES THIS APP DO?</div>
                <p>
                    This application finds the Pie Of The Day for each store and lists information about the store.<br/>
                    You can sort the Pies based on price, quantity of the pie of the day or by rating of the store.
                </p>
            </div>
            
        );
    }
}
export default About;