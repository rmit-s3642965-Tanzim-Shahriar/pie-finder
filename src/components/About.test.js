import React from 'react';
import { shallow } from 'enzyme';
import About from './About';
//WARNING: MUST comment out import './about.scss' from About.js to run this test
//as jest cant read .scss  
describe('About', () => {
  it('should render without any props', () => {
    const component = shallow(<About/>);
    expect(component).toMatchSnapshot();
  });
});