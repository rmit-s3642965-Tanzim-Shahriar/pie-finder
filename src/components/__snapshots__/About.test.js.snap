// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render without any props 1`] = `
<div
  className="about"
>
  <div
    className="headerTitle"
  >
    WHAT DOES THIS APP DO?
  </div>
  <p>
    This application finds the Pie Of The Day for each store and lists information about the store.
    <br />
    You can sort the Pies based on price, quantity of the pie of the day or by rating of the store.
  </p>
</div>
`;
